react/utils/metrics/createQuote.ts

```diff
@@ -67,12 +62,10 @@ const fetchMetricsData = async (
     throw new Error('Graphql Errors when trying get quote and user data')
   }
 
-  const quoteResult = data?.getQuote as Omit<QuoteMetricsData, 'costId'>
-  const costId = (data?.getUserByEmail?.[0].costId ?? '') as string
+  const quoteResult = data?.getQuote as QuoteMetricsData
 
   return {
     ...quoteResult,
```

Comment:
Since we don't have the `costId` anymore, maybe we could just do `return quoteResult` for better readability.

Reply:
Good point! Updated